frontend/src/features/map/overlays/VesselTrackCardOverlay.js

```diff
@@ -44,7 +44,7 @@ const VesselTrackCardOverlay = ({ map, feature }) => {
 
   useEffect(() => {
     if (overlayRef.current && overlayObjectRef.current) {
```

Comment:
Same thing here (it's almost the same code as `frontend/src/features/map/overlays/TrackTypeCardOverlay.js:36`).